msg/estimator_status.msg

```diff
@@ -13,38 +13,40 @@ float32[24] covariances	# Diagonal Elements of Covariance Matrix
+uint8 GPS_CHECK_FAIL_MIN_SAT_COUNT = 1		# 1 : minimum required sat count fail
+uint8 GPS_CHECK_FAIL_MIN_GDOP = 2		# 2 : minimum required GDoP fail
+uint8 GPS_CHECK_FAIL_MAX_HORZ_ERR = 3		# 3 : maximum allowed horizontal position error fail
+uint8 GPS_CHECK_FAIL_MAX_VERT_ERR = 4		# 4 : maximum allowed vertical position error fail
+uint8 GPS_CHECK_FAIL_MAX_SPD_ERR = 5		# 5 : maximum allowed speed error fail
+uint8 GPS_CHECK_FAIL_MAX_HORZ_DRIFT = 6		# 6 : maximum allowed horizontal position drift fail - requires stationary vehicle
+uint8 GPS_CHECK_FAIL_MAX_VERT_DRIFT = 7		# 7 : maximum allowed vertical position drift fail - requires stationary vehicle
+uint8 GPS_CHECK_FAIL_MAX_HORZ_SPD_ERR = 8	# 8 : maximum allowed horizontal speed fail - requires stationary vehicle
+uint8 GPS_CHECK_FAIL_MAX_VERT_SPD_ERR = 9	# 9 : maximum allowed vertical velocity discrepancy fail
 
 uint32 control_mode_flags	# Bitmask to indicate EKF logic state
-# 0 - true if the filter tilt alignment is complete
-# 1 - true if the filter yaw alignment is complete
-# 2 - true if GPS measurements are being fused
-# 3 - true if optical flow measurements are being fused
-# 4 - true if a simple magnetic yaw heading is being fused
-# 5 - true if 3-axis magnetometer measurement are being fused
-# 6 - true if synthetic magnetic declination measurements are being fused
-# 7 - true when the vehicle is airborne
-# 8 - true when wind velocity is being estimated
-# 9 - true when baro height is being fused as a primary height reference
-# 10 - true when range finder height is being fused as a primary height reference
-# 11 - true when GPS height is being fused as a primary height reference
-# 12 - true when local position data from external vision is being fused
-# 13 - true when yaw data from external vision measurements is being fused
-# 14 - true when height data from external vision measurements is being fused
-# 15 - true when synthetic sideslip measurements are being fused
-# 16 - true when only the magnetic field states are updated by the magnetometer
-# 17 - true when the vehicle is operating as a fixed wing vehicle
-# 18 - true when the magnetomer has been declared faulty and is no longer being used
-# 19 - true when airspeed measurements are being fused
+uint8 CS_TILT_ALIGN = 0		# 0 - true if the filter tilt alignment is complete
+uint8 CS_YAW_ALIGN = 1		# 1 - true if the filter yaw alignment is complete
+uint8 CS_GPS = 2		# 2 - true if GPS measurements are being fused
+uint8 CS_OPT_FLOW = 3		# 3 - true if optical flow measurements are being fused
+uint8 CS_MAG_HDG = 4		# 4 - true if a simple magnetic yaw heading is being fused
+uint8 CS_MAG_3D = 5		# 5 - true if 3-axis magnetometer measurement are being fused
+uint8 CS_MAG_DEC = 6		# 6 - true if synthetic magnetic declination measurements are being fused
+uint8 CS_IN_AIR = 7		# 7 - true when thought to be airborne
+uint8 CS_WIND = 8		# 8 - true when wind velocity is being estimated
```

Comment:
Unrelated to the PR, but is this flag for multirotor wind estimation or for fixed-wing?

Reply:
Both. If the multi rotor drag observation is enabled via the EKF2_AID_MASK parameter it would also be true for multi rotors.